content/docs/2.16/scalers/splunk_observability.md

```diff
@@ -0,0 +1,131 @@
++++
+title = "Splunk Observability"
+availability = "v2.15+"
```

Comment:
This would be 2.16+ assuming the new scaler is merged.

Reply:
@sschimper-splunk looks like you put 2.6 instead of 2.16 by mistake

Reply:
Since the version isn't available yet, the build will fail. So you need to explicitly ignore the URL like this: https://github.com/kedacore/keda-docs/pull/1413/files#diff-9ad64044531d7d8c033064f11141586478b535147312374c24f8d2d997722889R7